compiler+runtime/test/jank/reader-macro/cpp/boolean/pass-boolean.jank

```diff
@@ -0,0 +1,4 @@
+#cpp true
```

Comment:
All of these tests don't actually check the *usability* of the values. As such, they provide very little value. Yes, you copied the string tests, but those were poor to start with.

Put the value in a `let*` and only *conditionally* evaluate to `:success` if everything is correct. There are _many_ examples of this in this test suite.